### PR TITLE
UI changes and fuzzy search.

### PR DESCRIPTION
**Pull request by BrendinGreen**

### CODE REVIEW

File: app/assets/javascripts/map/map.controller.js.erb [hunk 1]
```diff
@@ -80,7 +80,7 @@ function MapController(DocumentService, $scope, $interval, retroMode, nightMode)
               width: 45,
               height: 45
           },
-          infowindow: '<div class="container-color-no-margin"><h5> Book Name: ' + book.name + '</h5><p> Description: ' + book.description + '</p><button class="btn btn-success" style="width: 100%" onclick="openBook(' + book.id + ')"> Open Book </button></div>'
+          infowindow: '<div class="container-color-no-margin"><h5> Book Name: ' + book.name + '</h5><p> Description: ' + book.description + '</p><button class="btn btn-warning" style="width: 100%; color: white" onclick="openBook(' + book.id + ')"> Open Book </button></div>'
```
Comment (digitalSalt) [inline app/assets/javascripts/map/map.controller.js.erb]: got a screen cap?

File: app/assets/javascripts/modal/modal.controller.js [hunk 2]
```diff
@@ -11,6 +11,34 @@ function ModalController($rootScope, DocumentService, $scope, UserService){
...
+
+    $ctrl.fuzzySearchOptions = {
+        shouldSort: true,
+        threshold: 0.6,
+        location: 0,
+        distance: 30,
+        maxPatternLength: 32,
+        minMatchCharLength: 1,
+        keys: [
+            "name",
```
Comment (digitalSalt) [inline app/assets/javascripts/modal/modal.controller.js]: should this be `filename` as documents have a filename attribute as opposed to `name` attribute?

File: app/assets/javascripts/map/map.controller.js.erb (hunk 1 above)
Comment (BrendinGreen) [reply app/assets/javascripts/map/map.controller.js.erb]: screen shot 2017-12-02 at 2 06 07 pm

File: app/assets/javascripts/modal/modal.controller.js (hunk 2 above)
Comment (BrendinGreen) [reply app/assets/javascripts/modal/modal.controller.js]: Absolutely correct

**Pull request merged**